src/core/server_irc_event.c

```diff
@@ -351,7 +351,7 @@ void server_irc_event_join(SircSession *sirc, const char *event,
     }
 
     if (youjoin) {
-        snprintf(buf, sizeof(buf), _("You has joined"));
+        snprintf(buf, sizeof(buf), _("You have joined"));
     } else {
         snprintf(buf, sizeof(buf), _("%1$s has joined"), origin);
```

Comment:
Here is a mistake too, can you please fix it in this PR?

Reply:
sorry, my fault, just ignore it.

Reply:
@SilverRainZ, if I'm not missing something it's fine here:
he/she/it - has;
you/we/they - have.